Lib/fontTools/varLib/featureVars.py

```diff
@@ -95,6 +96,10 @@ def addFeatureVariations(font, conditionalSubstitutions, featureTag="rvrn"):
 
     addFeatureVariationsRaw(font, font["GSUB"].table, conditionsAndLookups, featureTags)
 
+    # Recompute usMaxContext in case the font didn't have features before, but
+    # does now.
+    font["OS/2"].usMaxContext = maxCtxFont(font)
```

Comment:
Since we only add 1:1 features, maybe only set `usMaxContext` to the max of 1 and it's current value?

Also, I see you say `OS/2` table is required, but we deal with all kinds of non-OpenType-standard fonts all the time, so my preference would be to add a conditional while at it. Thanks.